databricks/koalas/plot.py

```diff
@@ -1186,181 +1186,168 @@ def _get_plot_backend(backend=None):
-        positional_args = {
-            ("ax", None),
-            ("figsize", None),
-            ("use_index", True),
-            ("title", None),
-            ("grid", None),
-            ("legend", False),
-            ("style", None),
-            ("logx", False),
-            ("logy", False),
-            ("loglog", False),
-            ("xticks", None),
-            ("yticks", None),
-            ("xlim", None),
-            ("ylim", None),
-            ("rot", None),
-            ("fontsize", None),
-            ("colormap", None),
-            ("table", False),
-            ("yerr", None),
-            ("xerr", None),
-            ("label", None),
-            ("secondary_y", False),
-        }
-    elif isinstance(data, DataFrame):
-        positional_args = {
-            ("ax", None),
-            ("subplots", None),
-            ("sharex", None),
-            ("sharey", False),
-            ("layout", None),
-            ("figsize", None),
-            ("use_index", True),
-            ("title", None),
-            ("grid", None),
-            ("legend", True),
-            ("style", None),
-            ("logx", False),
-            ("logy", False),
-            ("loglog", False),
-            ("xticks", None),
-            ("yticks", None),
-            ("xlim", None),
-            ("ylim", None),
-            ("rot", None),
-            ("fontsize", None),
-            ("colormap", None),
-            ("table", False),
-            ("yerr", None),
-            ("xerr", None),
-            ("secondary_y", False),
-            ("sort_columns", False),
-        }
-    # removing keys that are not required
-    kwargs.pop("self", None)
-    kwargs.pop("kwds", None)
-    kwargs.pop("kind", None)
-
-    for arg, def_val in positional_args:
-        # map the argument if possible
-        if backend_name in args_map:
-            if arg in kwargs and arg in args_map[backend_name]:
-                kwargs[args_map[backend_name][arg]] = kwargs.pop(arg)
-        # remove argument is default and not mapped
-        if arg in kwargs and kwargs[arg] == def_val:
-            kwargs.pop(arg, None)
-    return data_preprocessor_map[kind](data), kwargs
-
-
-class KoalasSeriesPlotMethods(PandasObject):
-    """
-    Series plotting accessor and method.
+    Series/Frames plotting accessor and method.
+
+    Uses the backend specified by the
+    option ``plotting.backend``. By default, matplotlib is used.
 
     Plotting methods can also be accessed by calling the accessor as a method
     with the ``kind`` argument:
     ``s.plot(kind='hist')`` is equivalent to ``s.plot.hist()``
     """
 
+    # from databricks.koalas import DataFrame, Series
+
+    _common_kinds = ("area", "bar", "barh", "box", "hist", "kde", "line", "pie")
+    _series_kinds = () + _common_kinds
+    _dataframe_kinds = ("scatter", "hexbin") + _common_kinds
+    _kind_aliases = {"density": "kde"}
+    _all_kinds = _common_kinds + _series_kinds + _dataframe_kinds
```

Comment:
The name `_all_kinds` could be ambiguous since it exists in this file.
Could you rename it?